### DISCUSSION TITLE
OtherInstitution table

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5609] [Feature Requests]
There should be an other institution table in Specify. This is important for historical purposes and to offer the ability to see all other institutions that have once handled a specimen or currently have a duplicate, etc.

Interactions could also be improved if we handled relationships through the institution rather than an individual in certain scenarios.

**Requested By:** New Mexico, West Virginia, French Guyane, and more herbaria.

**Comment by grantfitzsimmons** [comment]
RBGE would like a linkage to this table from the Agent, Collection Object, and Preparations tables.

Would be useful if the organization could have key people who work there. Assigning agents should be an option to the table.

Ability to link agents to this table would be a great addition.

**Requested By:** RBGE

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8208
